README.md

```diff
@@ -3,104 +3,83 @@
 - [Overview](#overview)
 - [Requirements](#requirements)
 - [Setup Instructions](#setup-instructions)
+  - [Configure Settings](#configure-settings)
   - [Docker Setup](#steps-using-docker)
   - [Run locally](#steps-to-run-directly)
-  - [Setup Virtual Env](#setup-virtual-env)
-  - [Configure Settings](#configure-settings)
-- [Starting all Processes](#starting-all-processes)
 - [Monitoring and Debugging](#monitoring-and-debugging)
+- [Setting up Slack workflow](#setting-up-slack-workflow)
 - [Architecture Details](#architecture-details)
 
 ## Overview
 
-![Audit-protocol workflow](docs/images/AuditProtocolOverview.png)
-Audit-Protocol is the component of a fully functional, distributed system that works alongside Pooler, off-chain consensus and together they are responsible for
+Architecture overview
+![Architecture](./docs/images/onchain_architecture.png)
+
+Audit-Protocol is the component of a fully functional, distributed system that works alongside Pooler and Epoch Generator together they are all responsible for
 * generating a time series database of changes occurring over smart contract state data and event logs, that live on decentralized storage protocols
 * higher order aggregated information calculated over decentralized indexes maintained atop the database mentioned above
 
-Audit-protocol can also be used for storing stream of json encoded data from any on-chain source(like a smart contract on a blockchain).
+Audit-protocol is used for committing snapshots to on-chain source (like a smart contract on a blockchain) via transaction relayer or directly to it.
 It has the following functions:
 
-1. [Store snapshots](https://www.notion.so/powerloom/PowerLoom-Protocol-Overview-c3bf9dd9323541118d46a4d8684565d1?pvs=4#68d50c3a7db14f2893706e805f083bec) on decentralized storage protocols like IPFS and Filecoin.
-2. Commit [anchor-proof](https://www.notion.so/powerloom/PowerLoom-Protocol-Overview-c3bf9dd9323541118d46a4d8684565d1?pvs=4#56c5333fa16940e8b0437e4d03129700) of snapshot on-chain
-2. Submit snapshots for [consensus](https://github.com/PowerLoom/product/issues/10)
-3. Track the [snapshot finalization](https://www.notion.so/powerloom/PowerLoom-Protocol-Overview-c3bf9dd9323541118d46a4d8684565d1?pvs=4#bf7a29dfd1384d28b45369b98d74a40f) and build a continuous DAG chain of snapshots
-4. Provide API to access the [DAG chain](https://www.notion.so/powerloom/PowerLoom-Protocol-Overview-c3bf9dd9323541118d46a4d8684565d1?pvs=4#c008da3137294ad6b065cfe0fcc05032). of snapshots
-5. Perform various high order operations such as time based indexing and aggregations on top of generated DAG chains
+<!-- TODO: add links to while paper for each of the functions below -->
+1. Store snapshots on decentralized storage protocols like IPFS and Filecoin.
+2. Submit snapshots to the on-chain source for consensus.
+3. Generate snapshotter report for snapshot submissions.
+4. Keep a local cache of finalized snapshots.
+5. Prune older snapshots.
```

Comment:
What exactly does `Prune older snapshots` mean?
I think we need to improve all 5 points and explain things in a better way. README should be easy to understand even for people who are not active contributors.

Reply:
Improved the  with better explanation